File: Word-Formed/src/com/google/corrigan/owen/wordformed/SinglePlayerGame.java
package com.google.corrigan.owen.wordformed;

import java.util.ArrayList;
import java.util.Map;

import android.app.Activity;
import android.content.Intent;
import android.media.AudioManager;
import android.os.Bundle;
import android.util.Log;
import android.view.Window;
import android.view.WindowManager;

public class SinglePlayerGame extends Activity
{
	private static final String TAG = SinglePlayerGame.class.getSimpleName();
	static ArrayList<Word> wordList;
	
	@Override
	public void onCreate(Bundle savedInstanceState)
	{
		super.onCreate(savedInstanceState);
		requestWindowFeature(Window.FEATURE_NO_TITLE);
		setVolumeControlStream(AudioManager.STREAM_MUSIC);
		setContentView(new SinglePlayerGameView(this));
		getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN, WindowManager.LayoutParams.FLAG_FULLSCREEN);		

		wordList = new ArrayList<Word>();
		//game.requestFocus();
	}
	
	@Override
	protected void onDestroy()
	{
		Log.d(TAG, "Destroyed");
		startActivity(new Intent(this, GameOverActivity.class));
		super.onDestroy();
	}
	
	protected void onStop()
	{
		Log.d(TAG, "Stopped");
		super.onStop();
	}
}

File: app/src/main/java/com/google/corrigan/owen/wordformed/SinglePlayerGame.kt
package com.google.corrigan.owen.wordformed

import android.media.AudioManager
import android.os.Bundle
import android.util.Log
import android.view.Window
import android.view.WindowManager
import androidx.appcompat.app.AppCompatActivity

class SinglePlayerGame : AppCompatActivity() {
    private val TAG = SinglePlayerGame::class.java.simpleName

    companion object {
        @JvmStatic
        var wordList: ArrayList<Word> = ArrayList()
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        supportRequestWindowFeature(Window.FEATURE_NO_TITLE)
        volumeControlStream = AudioManager.STREAM_MUSIC
        
        window.setFlags(
            WindowManager.LayoutParams.FLAG_FULLSCREEN,
            WindowManager.LayoutParams.FLAG_FULLSCREEN
        )

        setContentView(SinglePlayerGameView(this))
        
        wordList = ArrayList()
    }

    override fun onDestroy() {
        Log.d(TAG, "Destroyed")
        super.onDestroy()
    }

    override fun onStop() {
        Log.d(TAG, "Stopped")
        super.onStop()
    }
}
